index.yaml

```diff
@@ -393,14 +393,13 @@ entries:
     - https://github.com/longhorn/longhorn-engine
     - https://github.com/longhorn/longhorn-instance-manager
     - https://github.com/longhorn/longhorn-share-manager
-    - https://github.com/longhorn/backing-image-manager
     - https://github.com/longhorn/longhorn-manager
     - https://github.com/longhorn/longhorn-ui
     - https://github.com/longhorn/longhorn-tests
     - https://github.com/longhorn/backing-image-manager
     urls:
-    - assets/longhorn/longhorn-1.1.200+up1.1.2.tgz
-    version: 1.1.200+up1.1.2
+    - assets/longhorn-1.2/longhorn-1.2.0.tgz
+    version: 1.2.0
```

Comment:
The changes in this file are actually:
- add the 1.2.0 
- remove the 1.1.100+up1.1.1